[ 2024-08-08 15:17:19,454 ] 187 werkzeug - WARNING -  * Debugger is active!
[ 2024-08-08 15:17:19,482 ] 187 werkzeug - INFO -  * Debugger PIN: 144-597-497
[ 2024-08-08 15:17:25,891 ] 44 root - INFO - Entered initiate_data_ingestion method of Data_Integration class
[ 2024-08-08 15:17:25,892 ] 25 root - INFO - Exporting data to mongo db
[ 2024-08-08 15:17:25,893 ] 31 root - INFO - saving exported data into feature store file path: artifacts\08_08_2024_15_17_16\data_ingestion
[ 2024-08-08 15:17:25,893 ] 30 root - ERROR - Error in getting collection names: Error occurred python scipt name [D:\Data_Science\FSDS_Pro\Projects\phishing-classifier\src\configuration\mongo_db_connection.py] line number [21] error message [name 'nongo_db_url' is not defined]
[ 2024-08-08 15:17:25,894 ] 60 root - ERROR - Error in exporting collections as dataframe: Error occurred python scipt name [D:\Data_Science\FSDS_Pro\Projects\phishing-classifier\src\data_access\phishing_data.py] line number [26] error message [Error occurred python scipt name [D:\Data_Science\FSDS_Pro\Projects\phishing-classifier\src\configuration\mongo_db_connection.py] line number [21] error message [name 'nongo_db_url' is not defined]]
[ 2024-08-08 15:17:25,894 ] 40 root - ERROR - Error in exporting data into raw data directory: Error occurred python scipt name [D:\Data_Science\FSDS_Pro\Projects\phishing-classifier\src\data_access\phishing_data.py] line number [53] error message [Error occurred python scipt name [D:\Data_Science\FSDS_Pro\Projects\phishing-classifier\src\data_access\phishing_data.py] line number [26] error message [Error occurred python scipt name [D:\Data_Science\FSDS_Pro\Projects\phishing-classifier\src\configuration\mongo_db_connection.py] line number [21] error message [name 'nongo_db_url' is not defined]]]
[ 2024-08-08 15:17:25,894 ] 54 root - ERROR - Error in initiate_data_ingestion method: Error occurred python scipt name [D:\Data_Science\FSDS_Pro\Projects\phishing-classifier\src\components\data_ingestion.py] line number [33] error message [Error occurred python scipt name [D:\Data_Science\FSDS_Pro\Projects\phishing-classifier\src\data_access\phishing_data.py] line number [53] error message [Error occurred python scipt name [D:\Data_Science\FSDS_Pro\Projects\phishing-classifier\src\data_access\phishing_data.py] line number [26] error message [Error occurred python scipt name [D:\Data_Science\FSDS_Pro\Projects\phishing-classifier\src\configuration\mongo_db_connection.py] line number [21] error message [name 'nongo_db_url' is not defined]]]]
[ 2024-08-08 15:17:25,973 ] 187 werkzeug - INFO - 127.0.0.1 - - [08/Aug/2024 15:17:25] "[35m[1mGET /train HTTP/1.1[0m" 500 -
[ 2024-08-08 15:17:26,069 ] 187 werkzeug - INFO - 127.0.0.1 - - [08/Aug/2024 15:17:26] "[36mGET /train?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 304 -
[ 2024-08-08 15:17:26,070 ] 187 werkzeug - INFO - 127.0.0.1 - - [08/Aug/2024 15:17:26] "[36mGET /train?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 304 -
[ 2024-08-08 15:17:26,436 ] 187 werkzeug - INFO - 127.0.0.1 - - [08/Aug/2024 15:17:26] "[36mGET /train?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 -
[ 2024-08-08 15:17:26,819 ] 187 werkzeug - INFO - 127.0.0.1 - - [08/Aug/2024 15:17:26] "[36mGET /train?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 -
[ 2024-08-08 15:17:52,804 ] 187 werkzeug - INFO -  * Detected change in 'd:\\data_science\\fsds_pro\\projects\\phishing-classifier\\src\\configuration\\mongo_db_connection.py', reloading
